docs/decisions/0010-extract-enrollment-python-api.rst

```diff
@@ -0,0 +1,65 @@
+Extract python api from POST /api/enrollment/v1/enrollment handler
+==================================================================
+
+Status
+------
+Accepted
+
+
+Decisions
+---------
+
+We will extract some of the core enrollment functionality present in the POST handler from
+    `POST /api/enrollment/v1/enrollment`
+into a non-REST Python method while leaving functionality intact.
+
+This is so we can call the basic 'create enrollment in LMS' functionality from edx-enterprise without
+incurring REST expense. We also do not need to apply the rigorous access and embargo checks present
+in the api handler that check the `request` object, since the caller inthis case will be our co-located
+code running with the LMS already.
```

Comment:
Maybe a note in here that we're specifically creating this python method in the `enterprise_support` module.